portfolio/src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -30,6 +30,9 @@
   public static final Comparator<TimeRange> sortByTimeRangeStartTimeAscending =
       Comparator.comparing(TimeRange::start);
 
+  public static final Comparator<TimeRange> sortByTimeRangeDurationAscending =
```

Comment:
I see that this comparator is not used anywhere. And I can't think of a scenario when we will use it. I think we probably want to always sort the time range by start time. Can we remove this variable until we come across a case where we should use it?

Reply:
Updated. In the longest task first algorithm that I am also working on, I actually used a comparator that sorts time ranges by duration first and then start time, so I updated the constant variables for the comparator accordingly here.